packages/build/package.json

```diff
@@ -20,7 +20,9 @@
     "Swyx <swyx@netlify.com> (https://www.swyx.io)"
   ],
   "scripts": {
-    "prepublishOnly": "cd ../../ && npm run prepublishOnly"
+    "prepublishOnly": "cd ../../ && npm run prepublishOnly",
+    "prebuild": "rm -rf lib && mkdir lib",
+    "build": "npm run prebuild && cp -a src lib/src/ && cp package.json lib/"
```

Comment:
you should not need to add `npm run prebuild` here. pre scripts should be run automatically by npm. when run with lerna I guess you can chain then by adding the project dependencies in the json.

Why are you copying the package.json here as well? 

you should not have a `cp -a src lib/src` -a is the archive mode which does not preserve directories.

I would go with a classig `cp -R src lib` (don't nest the src folder inside the lib one)

This should be adapted on all packages